nano-theme.el

```diff
@@ -32,6 +32,14 @@
 
 (require 'nano-faces)
 
+(defcustom nano-theme-var nil
```

Comment:
Don't you need to set a default value (light)?

Reply:
I would like to set a default value, but since nano-theme can be used separately from the -dark and -light variants I was thinking it would be better to have you set this manually, as `require`ing (nano-theme-dark/light) would still enable the theme, while if you did not require a theme it would break, as it would try to set a theme.

I realize that's wrong now because the toggle function is not called, but requiring nano-theme-light/dark will set `nano-theme-var` anyway so I don't think there's much harm in it. I could set it to default to light nonetheless if you prefer.

I think this toggling would work better if nano-theme-light/dark would be included (or at least required) by nano-theme without being called, as then the variable could just be used to set which theme you want at the start, but that would break other things so I'm not sure that's preferable.